.eslintrc

```diff
@@ -3,9 +3,13 @@
   "rules": {
     "jsx-a11y/anchor-is-valid": false,
     "import/no-cycle": [2, { "maxDepth": 1 }],
-    "react/forbid-prop-types": false
+    "react/forbid-prop-types": false,
+    "react/no-unknown-property": false,
+    "react/jsx-no-bind": false,
+    "react/destructuring-assignment": false,
+    "react/no-array-index-key": false
```

Comment:
Disabling lint rules instead fixing the issues is not best practice.

1. Are easy to fix and affect just some files:
```
"react/jsx-no-bind": false,
"react/destructuring-assignment": false,
"react/no-array-index-key": false
```
2. This affect a lot of files and need some work to update all.
```
"react/no-unknown-property": false,
```
-----

I think these issue coming with bump from React version.

Reply:
cc @jmgasper

Reply:
@luizrrodrigues - Ok, can we approve this request, and then I'll get this fixed for the next build please?  I don't have time to fix this for our scheduled deploy today.

Reply:
Hey @jmgasper sure, let move forward with that and track these issues.

Please share the Jira ticket to track this.

Reply:
https://topcoder.atlassian.net/browse/PROD-4153